hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -522,21 +537,42 @@ else:
       .tl_h_o (tl_${m["name"]}_h_h2d),
       .tl_h_i (tl_${m["name"]}_h_d2h),
     % endif
-    % for p_in in m["available_input_list"] + m["available_inout_list"]:
-      % if loop.first:
+    % if m["type"] == "usbdev":
+
+      // Differential data
+      .cio_d_i          (1'b0),
+      .cio_d_o          (),
```

Comment:
Similar to the question above, does `cio_d_i` necessary? we could comment it out until we find the solution..

Reply:
It think is necessary to drive a defined value on the unused input. I have added a comment that the differential data signals are currently not used.